packages/ui/src/components/modal/modal.test.tsx

```diff
@@ -1,10 +1,10 @@
 import "@testing-library/jest-dom";
 import { fireEvent, render, screen } from "@testing-library/react";
```

Comment:
nit. Instead of `fireEvent`, it might be better to use the [`user-event` library](https://github.com/testing-library/user-event). The differences are described [here](https://testing-library.com/docs/user-event/intro/#differences-from-fireevent).